.github/workflows/pr-workflow.yaml

```diff
@@ -6,7 +6,7 @@ jobs:
     name: Mabl Test
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@main
+      - uses: actions/checkout@v3
```

Comment:
Are we opting to pin all actions to a specific version in general (i.e. across other repos as well)?

Reply:
Good question. The value here of pinning to a floating version tag is that we don't need to worry about pulling in an inadvertent breaking major version.